staging/src/k8s.io/apiserver/pkg/registry/rest/rest.go

```diff
@@ -209,6 +209,13 @@ type NamedCreater interface {
 	Create(ctx context.Context, name string, obj runtime.Object, createValidation ValidateObjectFunc, options *metav1.CreateOptions) (runtime.Object, error)
 }
 
+// SubresourceCreater adds configuration options to a Creater for subresources.
+type SubresourceCreater interface {
+	// PreserveObjectMetaSystemFields indicates that a handler should preserve
+	// fields of ObjectMeta that are managed by the system.
+	PreserveObjectMetaSystemFields() bool
```

Comment:
The marker method doesn't indicate this is related to subresources, or create requests, or where the data is being preserved. It's *very* verbose, but maybe make this as specific as possible (include the `subresource` aspect, the `create` aspect, and the fact that the _request_ data is what is being preserved)

`PreserveRequestObjectMetaSystemFieldsOnSubresourceCreate`